code.json

```
@@ -0,0 +1,45 @@
+[
+  {
+    "name": "drb-inland-salinity-ml",
+    "organization": "U.S. Geological Survey",
+    "description": "Code repo for Delaware River Basin machine learning models that predict inland salinity.",
+    "version": "main",
+    "status": "Development",
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://raw.githubusercontent.com/USGS-R/drb-inland-salinity-ml/main/LICENSE.md"
+        }
+      ]
+    },
+    "homepageURL": "https://github.com/USGS-R/drb-inland-salinity-ml/",
+    "downloadURL": "https://github.com/USGS-R/drb-inland-salinity-ml/archive/refs/heads/main.zip",
+    "disclaimerURL": "https://raw.githubusercontent.com/USGS-R/drb-inland-salinity-ml/main/DISCLAIMER.md",
+    "repositoryURL": "https://github.com/USGS-R/drb-inland-salinity-ml.git",
+    "vcs": "git",
+    "laborHours": -1,
+    "tags": [
+      "R",
+      "USGS",
+      "usg-artificial-intelligence",
+      "research",
+      "random forest",
+      "explainable AI",
+      "specific conductivity",
+      "rivers"
```

I think the tags you have are great, so up to you whether to add the suggested text below. It looks like `usg-artificial-intelligence` tag is a known or pre-existing tag? If so, does adding that tag come with any additional guidance or compliance requirements for our repo?

```suggestion
      "salinity",
      "river"
```

---

OK, thanks! Good to know about that `usg-artificial-intelligence` tag. I missed that when looking over the guidance before.